### ISSUE TITLE
Fig autocomplete not working (m1 macbook pro)

### ISSUE THREAD

**Issue by Mohamed-Motaz** [withfig/fig#2613]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.4.1 (22F82)

### Expected behaviour

I expect autocomplete to load suggestions

### Actual behaviour

The 3 dots appear for almost 10 seconds, and then no autocomplete suggestions appear, so the 3 dots disappear as well.

### Steps to reproduce

Run "cd " and expect a list of directories to appear. Attached below is an example of it running alongside the fig autocomplete dashboard, which I believe should provide hints

https://github.com/withfig/fig/assets/53558209/d659f6ad-82c0-4f2a-8853-3e87000c92a1

### Environment

```yaml
fig-details:
  - 2.16.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Max
  - cores: 10
  - mem: 64.00 GB
os-info:
  - macOS 13.4.1 (22F82)
environment:
  - shell: /bin/zsh
  - terminal: terminal
  - cwd: /Users/mohamedmotaz
  - exe-path: /Users/mohamedmotaz/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 30fc8d72-85af-4720-8b7c-ceeef3416586
    - FIG_SET_PARENT_CHECK: 1
    - FIG_TERM: 2.16.0
    - PATH: /opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:~/.dotnet/tools:/Library/Apple/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Users/mohamedmotaz/.fig/bin:/Users/mohamedmotaz/.local/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.apple.Terminal
    - FIG_PID: 59981
    - FIG_SET_PARENT: 30fc8d72-85af-4720-8b7c-ceeef3416586
```

**Comment by MooseSaeed** [comment]
Same

**Comment by HossamHoussien-adeva** [comment]
If you're in Egypt, the cdn domains are blocked (e.g.`cdn.jsdelivr.net`), and `fig` is relying on some JS files hosted on `jsdelivr`.

#### Without Cloudflare WARP (It will timeout after a few seconds, then failing)
<img width="952" alt="image" src="https://github.com/withfig/fig/assets/95684011/05d59092-8172-48e7-b965-cab2c1871268">

#### With Cloudflare WARP
<img width="952" alt="image" src="https://github.com/withfig/fig/assets/95684011/fbeb453c-40ce-448b-ae4b-8544bab2a588">

**Comment by mysticole928** [comment]
I have the same issue.  MacBook M1 Pro chip with fig integrations with both iTerm2 and VScode.  

Ran `fig doctor` to repair in both programs.

Neither `TAB` nor `ENTER` work.  I have to hit space a few times and then use a backspace to clear the fig menu to go away.

**Comment by Mohamed-Motaz** [comment]
@HossamHoussien-adeva Thanks, Cloudflare Warp did fix it. For anybody in the future, make sure the url fig points to isn't blocked by any third part entity, or use DNS 1.1.1.1, it may fix it.
